### ISSUE TITLE
Search: load more doesn't work right

### ISSUE THREAD

**Issue by Vistaus** [baedert/corebird#666]
When I search for something and then click on Load more, it starts to load the exact same accounts instead of other related accounts.

(for some reason Github was a bit buggy during the time of writing so I had to upload the screenshots elsewhere)
https://s6.postimg.org/513svjy3l/Screenshot_from_2017_01_10_20_44_48.png
https://s6.postimg.org/msffa0di9/Screenshot_from_2017_01_10_20_44_36.png

Using Corebird git master on Solus (Linux).

**Comment by baedert** [comment]
iirc that happens when there are exactly 3 users to show in the users part of the list. When we then  query the next 3, the Twitter API returns the same 3 instead of nothing.

**Comment by Vistaus** [comment]
You're right that there are not many users being found with those search terms but Twitter Mobile shows 4 instead of 3 with that same query. So there must be a bug in either the Twitter API or Corebird...

![screenshot from 2017-01-12 09-49-53](https://cloud.githubusercontent.com/assets/1716229/21882867/b11ef1e4-d8ac-11e6-81af-51b159f82304.png)

**Comment by baedert** [comment]
Yes, we query 4 entries, then show 3 so we know (so the theory) that if we get more than 3 results, a "show more results" button makes sense, since the next query will return at least one new result. We then get the same ones again unfortunately. (One could argue that not showing the fourth entry is a bug as well but not really the most disturbing one...).

**Comment by Vistaus** [comment]
I didn't say it's a disturbing bug :wink: But I would like to see if fixed. It doesn't have to have priority but a fix one day in the future would be welcomed :smile_cat:
